src/components/header.js

```diff
@@ -262,3 +264,10 @@ export default function Header(props) {
     </div>
   );
 }
+
+
+const mapState = state => ({
+  auth: state.auth
```

Comment:
do you need more than `isAuth`? by returning the minimal set of primitive props, redux is able to optimise renders better... if something else in `state.auth` changes, you don't need re-render the component.

Reply:
I'll note this down for the future! but for now auth only contains a boolean.